### PR TITLE
Use nginx-ingress-controller-app 1.3.0

### PR DESCRIPTION
**Pull request by sslavic**
Towards giantswarm/giantswarm#8664

### CODE REVIEW

File: pkg/project/version_bundle_azure.go [hunk 1]
```diff
@@ -7,11 +7,19 @@ import (
 var versionBundleAzure = versionbundle.Bundle{
 	Changelogs: []versionbundle.Changelog{
 		{
-			Component:   "cluster-operator",
-			Description: "TODO",
+			Component:   "nginx-ingress-controller",
+			Description: "Supported enabling the proxy protocol with AWS ELB.",
```
Comment (sslavic) [inline pkg/project/version_bundle_azure.go]: Not sure should this AWS relevant change be mentioned in Azure changelog

File: pkg/project/version_bundle_kvm.go [hunk 2]
```diff
@@ -7,11 +7,19 @@ import (
 var versionBundleKVM = versionbundle.Bundle{
 	Changelogs: []versionbundle.Changelog{
 		{
-			Component:   "cluster-operator",
-			Description: "TODO",
+			Component:   "nginx-ingress-controller",
+			Description: "Supported enabling the proxy protocol with AWS ELB.",
```
Comment (sslavic) [inline pkg/project/version_bundle_kvm.go]: Not sure should this AWS relevant change be mentioned in KVM changelog

File: pkg/project/version_bundle_azure.go (hunk 1 above)
Comment (MarcelMue) [reply pkg/project/version_bundle_azure.go]: Can be omitted for both non affected providers IMO. An upgrade notice is in there - that should be sufficient.

Comment (sslavic) [reply pkg/project/version_bundle_azure.go]: Thanks, I removed it.

File: pkg/project/version_bundle_aws.go [hunk 3]
```diff
@@ -7,11 +7,19 @@ import (
 var versionBundleAWS = versionbundle.Bundle{
 	Changelogs: []versionbundle.Changelog{
 		{
-			Component:   "cluster-operator",
-			Description: "TODO.",
+			Component:   "nginx-ingress-controller",
+			Description: "Supported enabling the proxy protocol with AWS ELB.",
```
Comment (sslavic) [inline pkg/project/version_bundle_aws.go]: Mentioned in AWS changelog only, since change is relevant for AWS only.

**Comment by sslavic** [comment]
Tested on giraffe

File: pkg/project/version_bundle_kvm.go [hunk 4]
```diff
@@ -22,7 +22,7 @@ var versionBundleKVM = versionbundle.Bundle{
 		},
 		{
 			Name:    "nginx-ingress-controller",
-			Version: "0.27.1",
+			Version: "0.28.0",
```
Comment (sslavic) [inline pkg/project/version_bundle_kvm.go]: @MarcelMue considering that the change is bump of nginx-ingress-controller, should this go into patch 0.22.1 or start next legacy cluster-operator minor 0.23.0?

Comment (MarcelMue) [reply pkg/project/version_bundle_kvm.go]: Let's do a minor.

**Pull request merged**